spec/factories/users.rb

```diff
@@ -32,6 +32,7 @@
 
     after(:create) do |user|
       user.is_admin = false
+      user.is_disabled = false
       # save with bang cause we want change in DB and not just in object instance
       user.save!
```

Comment:
> save with bang cause we want change in DB and not just in object instance

that's not true...
from the [Rails docs](http://api.rubyonrails.org/v4.2.5/classes/ActiveRecord/Persistence.html#method-i-save):

> By default, `save` always run validations. If any of them fail the action is cancelled and **save returns false**.

and:

> With `save!` validations always run. If any of them fail **ActiveRecord::RecordInvalid gets raised.**

(emphasis mine)